src/interfaces.py

```diff
@@ -15,26 +15,26 @@
 import devices.ph_probe as ph_probe  # pH
 import devices.ph_probe_mock as ph_probe_mock
 import devices.serial_mock as serial_mock
-import devices.temp_probe as temp_probe
-import devices.temp_probe_mock as temp_probe_mock
-import devices.tempcontrol as tempcontrol  # Temp
-import devices.tempcontrol_mock as tempcontrol_mock
+import devices.temperature_probe as temperature_probe
+import devices.temperature_probe_mock as temperature_probe_mock
+import devices.temperaturecontrol as temperaturecontrol  # Temperature
+import devices.temperaturecontrol_mock as temperaturecontrol_mock
 
 ph_class: types.ModuleType
-temp_class: types.ModuleType
+temperature_class: types.ModuleType
 board_class: types.ModuleType
 lcd_class: types.ModuleType
 keypad_class: types.ModuleType
-tempcontrol_class: types.ModuleType
+temperaturecontrol_class: types.ModuleType
 serial_class: types.ModuleType
 
 if constants.IS_TEST:
     ph_class = ph_probe_mock
-    temp_class = temp_probe_mock
+    temperature_class = temperature_probe_mock
     board_class = board_mock
     lcd_class = lcd_mock
     keypad_class = keypad_mock
-    tempcontrol_class = tempcontrol_mock
+    temperaturecontrol_class = temperaturecontrol_mock
```

Comment:
update to temperature_control_class

Reply:
I think I made a mistake while pushing the code, I pushed the wrong branch or file. I am working on it